### ISSUE TITLE
"Intro to Bioinformatics with R" by Lisa N. Cao

### ISSUE THREAD

**Issue by lfunderburk** [sciprog-sfu/sciprog-sfu.github.io#189]
## Description

This workshop covers an introduction to Bioinformatics with R. In this workshop, participants will have an opportunity to learn about packages such as Bioconductor https://www.bioconductor.org/ as well as SeqnR.

## Requirements

The following are instructions on how to set up your environment in preparation for the Introduction to Bioinformatics with R workshop, please save some space (let's say 50mb) for datasets.

Requires: R version 3.5.0 (version can be changed if multiple versions are installed locally, but takes awhile to download so please have it ahead of time), RStudio. It is also handy to have a text editor that is good for code such as emacs or notepad++.

Environment Setup:
Bioconductor official install guide: https://www.bioconductor.org/install/

#### install bioconductor base and dependencies
source("https://bioconductor.org/biocLite.R")

#### update bioconductor base
biocLite()

##### if you are having compiling issues try
install.packages("http://download.r-forge.r-project.org/bin/windows/contrib/latest.old/seqinr_3.4-6.zip", repos=NULL)

## Time and Place

**Where:** Room 7200, Library Research Commons, SFU Burnaby Campus

**When:** Tuesday, September 25, 2018 at 3:00 PM

## Lesson

https://github.com/lisancao/Bioinformatics_with_R

**Comment by lisancao** [comment]
Hi Laura @lfunderburk, do you mind assigning this issue to me so I can begin adding some materials? I was going to do a more basic workshop but will probably do Intro to Bioinformatics with R instead. Thanks!

**Comment by lfunderburk** [comment]
Hello @lisancao I tried adding you as an assignee but for some reason I am not able to add your id. Please let me know if you are able to edit directly the workshop description box. 

![gitissue](https://user-images.githubusercontent.com/13559004/45181664-f287c500-b1d3-11e8-8cd6-7e6380bcde14.png)

**Comment by prosoitos** [comment]
@lfunderburk You can't assign issues to people who are not part of the `SciProg` team (since they are not project owners). So @lisancao should send you the information by email and you will then be able to update the issue with that information.

**Comment by lisancao** [comment]
Hi @lfunderburk,
For this workshop people will need to have the latest version of R (3.5.1), Rstudio (strongly preferable), and the Bioconductor package installed. (https://www.bioconductor.org/install/). Tidyverse is also highly recommended. There may also be some other packages depending on whether or not I decide try to fit in some components like importing a dataset directly from NCBI or going over some visualizations and analyses. Will post here in the next few days if I do. I will be around at 2:30pm to help anyone with setting up their environment. If there's not enough time for importing data then I will upload some datasets for people to download from my github ahead of time or at the beginning of the talk while I am doing some background on the subject.

**Comment by lfunderburk** [comment]
Thank you for providing prerequisites on your workshop @lisancao 

We have booked the room from 3 to 4:30. Participants usually arrive about 5 minutes before to settle in. We can also ask people to setup in advance to make the most out of the booked time. If you are concerned about people not setting up prior to the workshop, one approach would be to set the first 15 minutes to ensure everyone is on the same page. 

We are currently adding descriptions and submitting to Research Commons so that the SFU community is aware of your workshop. If possible, please submit:

1) A short description of what the workshop entails - one or two sentences summarizing your workshop is fine, 
2) Prerequisites, i.e. a laptop with R (3.5.1) and Rstudio pre-installed along with packages: Bioconductor, Tidyverse, etc. 

**before September 13** to add to the Research Commons description. You can add this information on the comments and I will add it in the description. 

Let me know if you would like to change the title and also how I can support your workshop. 

Thanks again for stepping forward! :)

Laura

**Comment by lisancao** [comment]
Hi @lfunderburk I got the R version wrong, it's 3.5.0 since Bioconductor hasn't updated yet.

In this workshop we will go over the Bioconductor package for R, explore genomic datasets and do some preliminary analyses. Some basic R and genomic background is encouraged but not necessary to participate. 

===
The following are instructions on how to set up your environment in preparation for the Introduction to Bioinformatics with R workshop, please save some space (let's say 50mb) for datasets. 

Requires: R version 3.5.0 (version can be changed if multiple versions are installed locally, but takes awhile to download so please have it ahead of time), RStudio. It is also handy to have a text editor that is good for code such as emacs or notepad++. 

Environment Setup: 
Bioconductor official install guide: https://www.bioconductor.org/install/

##install bioconductor base and dependencies 
source("https://bioconductor.org/biocLite.R")

##update bioconductor base 
biocLite()

##Additional packages in Bioconductor (for personal interest only): https://www.bioconductor.org/packages/release/BiocViews.html#___Software

##Install package SeqnR instructions http://r-forge.r-project.org/R/?group_id=220
#if you are having compiling issues try 
install.packages("http://download.r-forge.r-project.org/bin/windows/contrib/latest.old/seqinr_3.4-6.zip", repos=NULL)

=======================

**Comment by lfunderburk** [comment]
Hello @lisancao 

Let me know how I can support you in tomorrow's workshop. 

I have class at 2:30 but I can meet you earlier during the afternoon to setup, get to the place, etc. 

Laura

**Comment by lisancao** [comment]
Hi Laura,

Sounds good. I will need you to let me into the room since I don't have the
code, but can meet you at the library entrance after you finish your class?
Is that alright? I have sticky notes I can bring.

Until the next,
Lisa N. Cao

**Comment by lfunderburk** [comment]
Hello Lisa, 

I can meet you at the library entrance. Unfortunately my class runs from 2:30  - 3:30 so I would not be able to meet you after my class is over, but I can meet you before. 

What is your email address? I can forward it to Research Commons so they can get in touch with you. They have provided sticky notes in the past and can help with the room. 

Laura

**Comment by lisancao** [comment]
Hi Laura,

My apologies, I misunderstood. I can meet you before then if you'd like,
but 45 minutes early is a lot. Will there be no other organizers able to
make it? My email is lncao@sfu.ca, if you could forward them my information
that would be great. If not I don't mind coming in super early for setup.

Until the next,
Lisa N. Cao

**Comment by lisancao** [comment]
All workshop materials are available here: 
https://github.com/lisancao/Bioinformatics_with_R
